sycl/unittests/helpers/UrImage.hpp

```diff
@@ -187,19 +149,44 @@ class UrPropertySet {
     // Value must be an all-zero 32-bit mask, which would mean that no fallback
     // libraries are needed to be loaded.
     UrProperty DeviceLibReqMask("", Data, SYCL_PROPERTY_TYPE_UINT32);
-    insert(__SYCL_PROPERTY_SET_DEVICELIB_REQ_MASK, UrArray{DeviceLibReqMask});
+    insert(__SYCL_PROPERTY_SET_DEVICELIB_REQ_MASK, DeviceLibReqMask);
   }
 
   /// Adds a new array of properties to the set.
   ///
   /// \param Name is a property array name. See ur.hpp for list of known names.
   /// \param Props is an array of property values.
-  void insert(const std::string &Name, UrArray<UrProperty> Props) {
+  void insert(const std::string &Name, std::vector<UrProperty> Props) {
```

Comment:
Could we make this

```suggestion
  void insert(const std::string &Name, std::vector<UrProperty> &&Props) {
```

to avoid copies? Seems like most cases I see, the argument is moved to insert anyway. Same question for the other insert functions.

This is not performance critical code though.

Reply:
Applied the suggestion in d591fbfba4bfff629180894630cd8262a9d1197e